include/swift/Basic/LangOptions.h

```diff
@@ -239,6 +242,10 @@ namespace swift {
     /// Diagnose uses of NSCoding with classes that have unstable mangled names.
     bool EnableNSKeyedArchiverDiagnostics = true;
 
+    /// Diagnose switches over non-frozen enums that do not have catch-all
+    /// cases.
+    bool EnableNonFrozenEnumExhaustivityDiagnostics = false;
```

Comment:
This is for staging while we settle on the "unknown case" spelling, right? It'll go away then?

Reply:
It's mainly so I can land this part before SE-0192's review period ends, but yes.